level0/terraform.tfstate

```diff
@@ -0,0 +1,9 @@
+{
+  "version": 4,
+  "terraform_version": "1.2.7",
+  "serial": 8,
+  "lineage": "6119ce0e-a54a-afe1-2811-2e6bec611731",
+  "outputs": {},
+  "resources": []
+}
+
```

Comment:
This state does not contain anything. Pllease run terraform apply in level0 and after that commit the generated state file.

Also, you don't need to run terraform destroy. It is ok to keep resources from level0 - they will not cost anything

Reply:
thanks for the tip!

Reply:
the file got updated